### ISSUE TITLE
Interactive Tutorial - Creating a Cluster

### ISSUE THREAD

**Issue by thomasstorm** [kubernetes/website#24650]
When I run
`kubectl version`
after I started minikub, the server seems to not start:
`Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.3", GitCommit:"06ad960bfd03b39c8310aaf92d1e7c12ce618213", GitTreeState:"clean", BuildDate:"2020-02-11T18:14:22Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}`
`The connection to the server localhost:8080 was refused - did you specify the right host or port?`
The next tutorial steps don't work due to this.

**Comment by sftim** [comment]
@thomasstorm if you try again, are the symptoms still present? (that may have been a glitch)

/priority awaiting-more-evidence
/triage needs-information

**Comment by thomasstorm** [comment]
@sftim In fact, now it works! Thanks. Downside is that the CSS cannot be loaded anymore on the tutorial page -- shall I create another issue for that? See screenshot below. Strange dependency anyways ;-)

![image](https://user-images.githubusercontent.com/213119/96693386-79359700-1387-11eb-9db1-c973df6d1ddf.png)

**Comment by niklasR** [comment]
Hi, I'm having the same issue as @thomasstorm, and I think it is due to my adblock (uBlock Origin)
After step 2, the terminal is running minikube, but it's not actually launching Kubernetes:
![image](https://user-images.githubusercontent.com/724977/97083032-8069d880-1605-11eb-8d96-e6ef27190923.png)

Disabling uBlock Origin starts kubernetes properly.
